openedx/core/djangoapps/theming/core.py

```diff
@@ -21,9 +22,14 @@ def enable_theming():
         )
 
     for theme in get_themes():
-        locale_dir = theme.path / "conf" / "locale"
-        if locale_dir.isdir():
-            settings.LOCALE_PATHS = (locale_dir, ) + settings.LOCALE_PATHS
-
```

Comment:
Why did you decide to append instead of prepending? Such a solution does not let us override any translation in theme (that's required for languages with poor rewiewing).